app/manager-frontend/src/components/Logo.jsx

```diff
@@ -0,0 +1,209 @@
+import React from "react";
+export const HutchLogo = ({ logoColor, logoMaxWidth, logoFillColor }) => {
```

Comment:
Is it not possible to extract the SVG stuff to a separate file and style it like the bottom of the [this page](https://wendeehsu.medium.com/styling-svg-icon-in-react-js-bfd3d826901d)?

Reply:
Thanks for the link. It seems Vite doesn't allow importing SVG file as react component by default. Thus, I have used the plugin `vite-plugin-svgr` for that.

I have also moved the favicon folder containing relevant files inside the `public/assets` folder. Hutch logo (mono and color) SVGs are also inside `public/assets` folder